src/codegen_internal.h

```diff
@@ -1,6 +1,6 @@
 // This file is a part of Julia. License is MIT: http://julialang.org/license
 
-#if defined(LLVM38) && !defined(LLVM37)
+#if JL_LLVM_VERSION >= 30800
```

Comment:
The condition there before was `false`, the current condition is wrong. The correct one is in `jitlayers.cpp` and is the same with the one I gave above.